src/CaptureFile/CaptureFile.cpp

```diff
@@ -259,6 +261,9 @@ std::optional<uint64_t> orbit_capture_file::CaptureFileImpl::FindSectionByType(
 
   return std::nullopt;
 }
+const std::filesystem::path& orbit_capture_file::CaptureFileImpl::GetFilePath() const {
```

Comment:
```suggestion
}

const std::filesystem::path& CaptureFileImpl::GetFilePath() const {
```

Reply:
You don't need to specify `orbit_capture_file::`, right? (Also `FindSectionByType` right above)

Reply:
yep - clion always does it to me for some reason.

Reply:
I feel you.

Reply:
done.